src/EditorFeatures/Test2/FindReferences/FindReferencesTests.OrdinaryMethodSymbols.vb

```diff
@@ -54,6 +54,54 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.FindReferences
         End Function
 
         <WpfFact, Trait(Traits.Feature, Traits.Features.FindReferences)>
+        <WorkItem(18963, "https://github.com/dotnet/roslyn/issues/18963")>
+        Public Async Function FindReferences_Deconstruction() As Task
+            Dim input =
+<Workspace>
+    <Project Language="C#" CommonReferences="true">
+        <Document>
+class C
+{
+    public void {|Definition:Decons$$truct|}(out int x1, out int x2) { x1 = 1; x2 = 2; }
+    public void M()
+    {
+        [|var (x1, x2) =|] this;
```

Comment:
Fasscinating.  This is the resultant span?  I guess i'm ok with that.  though i think it's strange.

Reply:
Yeah, I think something a bit narrower might be wise. Consider if there are comments in the middle of that somewhere...